### DISCUSSION TITLE
Accounts Module

### DISCUSSION THREAD

**Discussion by Akaki-1991** [YetiForceCompany/YetiForceCRM#15527] [General]
hello, I have a weird case in accounts module, I can not register the record, but when I turn off the duplicate, it registers after that.
![Capture](https://user-images.githubusercontent.com/75428743/145171394-b45b2261-4f23-4dde-8bba-ac65af174f33.JPG)

**Comment by mariuszkrzaczkowski** [comment]
check logs
